src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const rowButtons = document.querySelectorAll('[class*="row"]');
+const columnButtons = document.querySelectorAll('[class*="column"]');
+const rows = [...table.querySelectorAll('tr')];
+const column = rows[0].cells;
+let rowsNumber = rows.length;
+let columnNumber = column.length;
+
+document.body.addEventListener('click', (e) => {
+  const button = e.target.closest('.button');
+
+  if (button) {
+    const buttonFunc = [...button.classList];
+    const regex = /column/g;
+    const containColumns = buttonFunc.some((cls) => regex.test(cls));
+
+    table.innerHTML = '';
+
+    if (containColumns) {
+      const appendRegex = /append/g;
```

Comment:
The 'appendRegex' is defined twice (lines 22 and 45) with the same value. Consider moving it outside of the if-else block and defining it only once to adhere to the DRY (Don't Repeat Yourself) principle.